android/src/test/java/org/mozilla/nimbus/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package org.mozilla.nimbus
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
@tarikeshaq i think it's also worth adding a README for the `android` directory. Mostly a HOWTO build locally I think

Reply:
Yes, any environment requirements or setup instructions would be useful.